corehq/apps/cloudcare/static/cloudcare/js/utils.js

```diff
@@ -186,7 +237,21 @@ hqDefine('cloudcare/js/utils', [
     };
 
     var hideLoading = function () {
-        NProgress.done();
+        hqRequire(["cloudcare/js/formplayer/app", "hqwebapp/js/toggles"], function (FormplayerFrontend, toggles) {
```

Comment:
Do you remember why you need `hqRequire` here instead of `hqImport`? Same question for the `hqRequire` below.

Reply:
I was initially using `hqImport` and that worked fine, but was getting a test failure. I believe it came from here https://github.com/dimagi/commcare-hq/blob/0acf35279639c695b943784704a9f74ce6a86465/corehq/apps/hqwebapp/tests/test_requirejs.py#L98

Reply:
Cool, makes sense. That error message ought to be more helpful: https://github.com/dimagi/commcare-hq/pull/34276